libyul/backends/evm/EVMCodeTransform.cpp

```diff
@@ -16,7 +16,7 @@
 */
 // SPDX-License-Identifier: GPL-3.0
 /**
- * Common code generator for translating Yul / inline assembly to EVM and EVM1.5.
```

Comment:
Also not so much related, but we removed EVM1.5 support a while ago and just didn't adjust the comment.